src/main/java/ieti/voicebox/controller/AudioBookController.java

```
@@ -16,46 +19,57 @@
 import ieti.voicebox.model.AudioBook;
 import ieti.voicebox.persistence.PersistenceException;
 import ieti.voicebox.service.AudioBookService;
+import org.springframework.web.bind.annotation.RequestParam;
 
+@CrossOrigin(maxAge = 3600)
 @Controller
-@RequestMapping("audioBooks")
+@RequestMapping("/audioBooks")
 public class AudioBookController {
-
+	
 	@Autowired
 	private AudioBookService audioBookService;
-
-	@PostMapping()
-	public ResponseEntity<?> createAudioBook(@RequestBody AudioBook audioBook) throws PersistenceException {
+	
+	@PostMapping
+	public ResponseEntity<?> createAudioBook(@RequestBody AudioBook audioBook){
+		System.out.println("entro ");
 		try {
 			audioBookService.createAudioBook(audioBook);
 		} catch (PersistenceException e) {
-			throw new PersistenceException(e.getMessage());
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

If you silence the persistence exception here, then why do you need to throw the exception?